rules/rate_limit.go

```diff
@@ -134,7 +134,7 @@ func (r *RateLimitEnforcer) HandleRequest(ctx context.Context, uri string,
 	// Determine the start time of the actions window.
 	startTime := time.Now().Add(
 		-time.Duration(rateLim.NumHours) * time.Hour,
-	)
+	).UTC()
```

Comment:
This should be:
```
	startTime := time.Now().UTC().Add(
		-time.Duration(rateLim.NumHours) * time.Hour,
	)
```

Else there can be an discrepancy when the local timezone conversion happens right at a boundary of `DST`.